Configured error handling with a try and except block,

Joeroble opened the pull request:
and checking if data exists.

It also does not use a dry exception, so I believe this will be appropriate, but please correct me if I'm wrong.

claraj reviewed (commented):
This needs documentation.  What should the caller expect to be returned if the request works; what should they expect to be returned if there is an error? How can they differentiate between a successful request that returned no data (nothing on the wikipedia page for that date) vs. an error making the request (no internet, wikipedia down)

* What data type is the  user_date? A string, if so what format should it be in? A date? something else?
* Why return data[3] ? 
*

Joeroble commented:
> This needs documentation. What should the caller expect to be returned if the request works; what should they expect to be returned if there is an error? How can they differentiate between a successful request that returned no data (nothing on the wikipedia page for that date) vs. an error making the request (no internet, wikipedia down)
> 
>     * What data type is the  user_date? A string, if so what format should it be in? A date? something else?
> 
>     * Why return data[3] 

Fair enough, my bad.

Joeroble commented:
Added documentation, and added some other things - such as returning the error raised, messages.

Joeroble commented:
Changed the long comment to a doc string.

Joeroble commented:
Changed the return for exceptions to false, removed a message from the None return - configured to be in line with the handling response and call in api_manager.

Joeroble commented:
This is the updated version, also what everyone else should standardize to!

claraj commented on API_Wiki_Call.py:
```diff
@@ -2,16 +2,37 @@
+
+            if wiki_data_check:
+                data = wiki_date_page_selection
+                wiki_api_response.data = data
```
Is the entire response in a usable format by the UI?

NoonBrew replied:
That is not quite clear at this point. Currently, I have a landing page with a date selector and a button. Once I have passing info from the date selector to the Movie API and displaying results from that one API we will then be able to meet as a group and go over how we want to pass and show the other API's results.

claraj commented on API_Wiki_Call.py:
```diff
@@ -2,16 +2,37 @@
-data = requests.get(url, params = query).json()
+    """ This will try to get the request from wiki, if it fails for some reason, a message will print, 
+     and return the error, api manager should handle errors from the API call.  Assuming it is successful, 
+     and there is an item now in wiki_date_page_selection, that will then be stored in extracted_page_url, and returned. 
```
Is this docstring still relevant? I don't see an `extracted_page_url` variable

claraj reviewed (commented):
Couple of notes for you - think about all the things that could go wrong and make sure your code is correctly indicating connection error vs unexpected JSON format error.
